cypress/e2e/dev/6-management-tests/create-new-page-from-template.cypress.js

```diff
@@ -36,39 +42,86 @@ describe('create new start page', () => {
       .should('contains.text', 'Start now')
   })
 
-  it('Create the start page from the management page', () => {
-    cy.task('log', 'The start page should not be found')
-    cy.visit(startPagePath, { failOnStatusCode: false })
-    cy.get('body')
-      .should('contains.text', `Page not found: ${startPagePath}`)
+  describe('Create the start page from the management page', () => {
+    const testCreatePage = (pagePath, pageView) => () => {
+      cy.task('log', 'The start page should not be found')
+      cy.visit(pagePath, { failOnStatusCode: false })
+      cy.get('body')
+        .should('contains.text', `Page not found: ${pagePath}`)
 
-    cy.task('log', 'Visit the manage prototype templates page')
-    cy.visit(manageTemplatesPagePath)
-
-    cy.get(`a[href="${getTemplateLink('install', 'govuk-prototype-kit', '/lib/templates/start.html')}"]`).click()
-
-    cy.task('log', 'Create the page')
-    cy.get('.govuk-heading-l')
-      .should('contains.text', 'Create new Start page')
-    cy.get('.govuk-label')
-      .should('contains.text', 'Path for the new page')
-    cy.get('#chosen-url')
-      .type(startPagePath)
-    cy.get('.govuk-button')
-      .should('contains.text', 'Create page').click()
-
-    cy.task('log', 'Confirmation page')
-    cy.get('.govuk-heading-l')
-      .should('contains.text', 'Page created')
-
-    cy.task('log', 'Confirm the page exists')
-    cy.get(`a[href="${startPagePath}"]`)
-      .should('contains.text', startPagePath).click()
-    cy.task('log', 'The start page should be displayed')
-    cy.get('.govuk-button--start')
-      .should('contains.text', 'Start now')
+      cy.task('log', 'Visit the manage prototype templates page')
+      cy.visit(manageTemplatesPagePath)
+
+      cy.get(`a[href="${getTemplateLink('install', 'govuk-prototype-kit', '/lib/templates/start.html')}"]`).click()
+
+      cy.task('log', 'Create the page')
+      cy.get('.govuk-heading-l')
+        .should('contains.text', 'Create new Start page')
+      cy.get('.govuk-label')
+        .should('contains.text', 'Path for the new page')
+      cy.get('#chosen-url')
```

Comment:
just wondering if we should use `path` throughout, unless we are sometimes dealing with urls?

Reply:
The headings on the page were not part of the PR.  The only reason they are referenced is to for the acceptance tests.  Should this be in another PR?